src/edgerc.js

```diff
@@ -59,17 +67,27 @@ function buildObj(configs) {
   var result = {},
     index,
     key,
-    val;
+    val,
+    isComment;
 
   configs.forEach(function(config) {
+    config = config.trim();
+    isComent = config.indexOf(";") === 0;
     index = config.indexOf('=');
-    key = config.substr(0, index);
-    val = config.substr(index + 1, config.length - index - 1);
-
-    // Remove trailing slash as if often found in the host property
-    val = val.replace(/\/$/, '');
-
-    result[key.trim()] = val.trim();
+    if (index > -1 && !isComment) {
+      key = config.substr(0, index);
+      val = config.substring(index + 1);
+      // remove inline comments
+      if (val.search(/["']/)) {
+        val = val.replace(/\s*["'](.+)["']\s*;?.*$/, "$1");
```

Comment:
Typically we'd run afoul of something like this: `"…\"… "` with the escaped quote not being ignored — however, we know the values in this file, and we should never see this.

Reply:
That was my reasoning too. Do you think I need to take that into account? This also behaves erratically if a string has mixed quotation symbols (eg `key = "quoted 'quoted' value"`). I will try to prevent these issues